test/fixtures/tracks/fake/config.json

```diff
@@ -2,6 +2,7 @@
   "slug": "fake",
   "language": "Fake",
   "repository": "https://github.com/exercism/xfake",
+  "checklist_issue": "1",
```

Comment:
Let's put something not-1 here, and then have a different fixture track without this, which we can default to 1.